crates/bevy_render/src/view/visibility/range.rs

```diff
@@ -79,6 +79,13 @@ impl Plugin for VisibilityRangePlugin {
 /// between levels, each level has a *margin*, within which the object
 /// transitions gradually from invisible to visible using a dithering effect.
 ///
+/// You can also use this feature to replace multiple meshes with a single mesh
+/// when the camera is distant—hence the term *hierarchical*. This can be useful
```

Comment:
```suggestion
/// when the camera is distant — hence the term *hierarchical*. This can be useful
```

This kinda renders as distant-hence, which is confusing.

Reply:
I just reworded it.